csharp/ql/src/API Abuse/IncorrectCompareToSignature.ql

```diff
@@ -40,6 +40,5 @@ where
   compareToMethod(m, actualParamType) and
   not implementsIComparable(declaringType, actualParamType)
 select m,
-  "The parameter of this 'CompareTo' method is of type $@, but $@ does not implement 'IComparable<$@>'.",
-  actualParamType, actualParamType.getName(), declaringType, declaringType.getName(),
-  actualParamType, actualParamType.getName()
+  "The parameter of this 'CompareTo' method is of type '" + actualParamType.getName() +
```

Comment:
Consider avoid repeating `actualParamType.getName()` by introducing a variable for it in `from`.